### PR TITLE
Update patch release of .Net Core 2.1 SDK.

### PR DESCRIPTION
**Pull request by ILMTitan**

### CODE REVIEW

**Comment by henkmollema** [comment]
When can we expect this image to be publicly available?

**Comment by henkmollema** [comment]
Forgive my impatience, but is there any timeline on this @ILMTitan? We would like to upgrade our applications to 2.1.1.

**Comment by ILMTitan** [comment]
@henkmollema Unfortunately, I am not in control of when this gets merged and released. I will ping someone who is.

**Comment by henkmollema** [comment]
Thanks!

**Comment by henkmollema** [comment]
In the meanwhile [.NET Core 2.1.2/2.1.302](https://github.com/dotnet/core/blob/master/release-notes/2.1/2.1.2.md) just shipped. Perhaps this PR should be updated as well. Thanks in advance!

**Comment by henkmollema** [comment]
@ILMTitan any update on this?

**Comment by bendory** [comment]
@iantalarico can you PTAL?

File: dotnet/cloudbuild.yaml [hunk 1]
```diff
@@ -29,8 +29,8 @@ steps:
 - id: 'test the 2.1 sdk'
   name: 'gcr.io/$PROJECT_ID/dotnet'
   entrypoint: 'bash'
-  args: ['-c', '[[ $(dotnet --version) == "2.1.300" ]]']
-  dir: 'configs/config-2.1.300'
+  args: ['-c', '[[ $(dotnet --version) == "2.1.302" ]]']
+  dir: 'configs/config-2.1.301'
```
Comment (iantalarico) [inline dotnet/cloudbuild.yaml]: 2.1.302?

Comment (ILMTitan) [reply dotnet/cloudbuild.yaml]: Sorry about that. Done.

**Pull request merged**